hyperscale/008-grant_usage_fdw.sql

```diff
@@ -0,0 +1,2 @@
+-- source https://msdata.visualstudio.com/Database%20Systems/_git/CitusMigration?path=/instance/lib/grant_usage_fdw.sql
```

Comment:
maybe don't include links to the internal repo in this public repo

Reply:
Removed links